lib/SimpleSAML/XML/Shib13/AuthnResponse.php

```diff
@@ -6,12 +6,23 @@
  * @author Andreas Åkre Solberg, UNINETT AS. <andreas.solberg@uninett.no>
  * @package SimpleSAMLphp
  */
-class SimpleSAML_XML_Shib13_AuthnResponse {
+
+namespace SimpleSAML\XML\Shib13;
+
+
+use SAML2\DOMDocumentFactory;
+use SAML2\Utils;
+use SimpleSAML\Utils\Config;
+use SimpleSAML\Utils\Random;
+use SimpleSAML\Utils\Time;
+use SimpleSAML\XML\Validator;
+
+class AuthnResponse {
 
 	/**
-	 * This variable contains an XML validator for this message.
+	 * @var Validator This variable contains an XML validator for this message.
```

Comment:
I believe here we should use the full name of the class, including the namespace. I've observed problems with documentation that doesn't do that, because the `use` clauses in the beginning of the file are ignored. Therefore, I think we should use:

```
 * @var SimpleSAML\XML\Validator ...
```